lambda/custom/index.js

```diff
@@ -528,5 +916,6 @@ exports.handler = skillBuilder
     UnhandledIntent
   )
   .addRequestInterceptors(LocalizationInterceptor)
+  .addResponseInterceptors(LocalizationInterceptor)
```

Comment:
why is this on the request and response?

Reply:
Added by mistake. Will remove